doc/dev/fluxing.rst

```diff
@@ -0,0 +1,116 @@
+.. include:: ../include/links.rst
+
+.. _fluxcalib:
+
```

Comment:
This document should be linked to the toctree somehow. Possibly from development as it's in the dev directory. Or maybe linked from fluxing.rst in the parent directory.